js/apps/viewer/src/app/animation-player/animation-speed-controller.tsx

```diff
@@ -0,0 +1,78 @@
+import React, {
+    Component,
+    MouseEvent
+} from 'react';
+
+import Button from '@material-ui/core/Button';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+
+const animationSpeedId = 'animation-speed-controller';
+
+function speedLabel(value: number) {
+    return `x${value}`;
+}
+
+export default class AnimationSpeedController extends Component<Props, State> {
+    state : State = {
+        anchorEl: null
+    }
+
+    openMenu = (evt : MouseEvent<HTMLElement>) => this.setState({ anchorEl: evt.currentTarget });
```

Comment:
Run `yarn test` at the root of the `js` folder to catch these code format issues.

```suggestion
    openMenu = (evt: MouseEvent<HTMLElement>) => this.setState({anchorEl: evt.currentTarget});
```